### DISCUSSION TITLE
Can't boot into recovery mode.

### DISCUSSION THREAD

**Discussion by jakitliang** [utmapp/UTM#5783] [Linux]
**Describe the issue**  
You can test it with:
`https://mac.getutm.app/gallery/debian-11-ldxe`
and also
`https://mac.getutm.app/gallery/debian-10-4-i3`

No one can boot into recovery mode.

Also you can install any other Debian / ubuntu and also other linux.

You can try grub add "init=/bin/bash" to linux line. Or something like rescue option to let kernel boot into recovery mode.

But unfortunately, UTM can't do this.

UTM Always stay on a black screen with blinking cursor

**Configuration**  
* UTM Version: Latest
* macOS Version: 12
* Mac Chip: M1

**Crash log**  
Nope

**Debug log**  
Nope

**Upload VM**  
https://mac.getutm.app/gallery/debian-10-4-i3

**Comment by tmtiwari** [comment]
Same issue with `UTM Version 4.4.2 (90)` and `Ubuntu 23.04` running on `macOS 13.6 - M1 Pro`

**Comment by DiegoBaldassarMilleuno** [comment]
Same problem here. For the posterity, a possible alternative is to instead append `systemd.debug_shell` to kernel parameters and switch to TTY9.

**Comment by johnzhou721** [comment]
Another alternative is to boot from an installation ISO file of the operating system.
